File: linda-core/src/main/java/it/unibo/coordination/linda/core/InspectableNegatedTupleSpace.java
package it.unibo.coordination.linda.core;

public interface InspectableNegatedTupleSpace<T extends Tuple, TT extends Template, K, V>
        extends NegatedTupleSpace<T, TT, K, V>, InspectableTupleSpace<T, TT, K, V> {

}

File: linda-core/src/main/kotlin/it/unibo/coordination/linda/core/InspectableNegatedTupleSpace.kt
package it.unibo.coordination.linda.core

interface InspectableNegatedTupleSpace<T : Tuple<T>, TT : Template<T>, K, V, M : Match<T, TT, K, V>>
    : NegatedTupleSpace<T, TT, K, V, M>,
        InspectableLindaTupleSpace<T, TT, K, V, M>